package.js

```diff
@@ -1,68 +1,68 @@
 Package.describe({
-  summary: "Foundation for Sites 5 with Scss",
-  version: "3.0.0",
-  name: "juliancwirko:zf5",
-  git: "https://github.com/juliancwirko/meteor-zf5.git",
+  summary: 'Foundation for Sites 5 with Scss',
+  version: '3.0.0-modified',
```

Comment:
Could you change version to 4.0.0 ? Thanks.

Reply:
done!